.github/NEW_RELEASE.md

```diff
@@ -0,0 +1,53 @@
+---
+name: New Release
+about: Propose a new release
+title: Release v0.x.0
+assignees: ahg-g, alculquicondor, ArangoGutierrez, denkensk
+
+---
+
+## Release Checklist
+<!--
+Please do not remove items from the checklist
+-->
+- [ ] All [OWNERS](https://github.com/kubernetes-sigs/kueue/blob/master/OWNERS) must LGTM the release proposal
+- [ ] Verify that the changelog in this issue is up-to-date
+- [ ] For major releases (v0.$MAJ.0) create new release branch
+  - [ ] an OWNER creates a vanilla release branch with
+        `git branch release-0.$MAJ master`
+  - [ ] An OWNER pushes the new release branch with
+        `git push release-0.$MAJ`
+- [ ] Update things like README, deployment templates, docs, configuration, test/e2e flags, submit a PR agains the release branch
+- An OWNER prepares a draft release
+  - [ ] Create a draft release at [Github releases page](https://github.com/kubernetes-sigs/kueue/releases)
+  - [ ] Write the change log into the draft release
+  - [ ] Upload release artefacts
+- [ ] An OWNER runs
+     `git tag -s $VERSION`
+      and inserts the changelog into the tag description.
+- [ ] An OWNER pushes the tag with
+      `git push $VERSION`
+  - Triggers prow to build and publish a staging container image
+      `gcr.io/k8s-staging-kueue/kueue:$VERSION`
+  - Triggers build of the documentation and publish it at
+        https://kubernetes-sigs.github.io/kueue
+- [ ] Submit a PR against [k8s.io](https://github.com/kubernetes/k8s.io), updating `k8s.gcr.io/images/k8s-staging-kueue/images.yaml` to promote the container images (both "full" and "minimal" variants) to production
+- [ ] Wait for the PR to be merged and verify that the image (`k8s.gcr.io/kueue/kueue:$VERSION`) is available.
+- [ ] Publish the draft release prepared at the [Github releases page](https://github.com/kubernetes-sigs/kueue/releases)
+      which will also trigger a Helm repo index update to add the latest release
+- [ ] Add a link to the tagged release in this issue.
+- [ ] Send an announcement email to `dev@kubernetes.io` with the subject `[ANNOUNCE] kueue $VERSION is released`
```

Comment:
just sig-scheduling and wg-batch should be enough

Reply:
done